libvcx/src/libindy/utils/wallet.rs

```diff
@@ -87,18 +97,15 @@ pub fn create_wallet_from_config(config: &str) -> VcxResult<()> {
     Ok(())
 }
 
-pub fn configure_issuer_wallet(enterprise_seed: &str) -> VcxResult<String> {
-
+pub fn configure_issuer_wallet(enterprise_seed: &str) -> VcxResult<IssuerConfig> {
     let (institution_did, institution_verkey) = signus::create_and_store_my_did(Some(enterprise_seed), None)?;
-
+    // todo: We sohuld probably not be setting the state here, but leave it up to caller to call "fn init_issuer_config()" subsequently after this function
     settings::set_config_value(settings::CONFIG_INSTITUTION_DID, &institution_did);
     settings::set_config_value(settings::CONFIG_INSTITUTION_VERKEY, &institution_verkey);
```

Comment:
That's correct, so do we actually have to set the values here?

Reply:
I think that's how original logic was and I didn't understand why. So I've deleted it now, lets see if we still pass tests.

Reply:
Awesome